src/styles/blocks/clock.scss

```
@@ -0,0 +1,96 @@
+@use "../utils/extends";
+@use "../utils/variables" as *;
+
+html {
+  font-size: 44px;
+  font-weight: 500;
+  font-family: "Roboto", sans-serif;
+}
+
+.body {
+  margin: 0;
+  height: 100vh;
+  background-color: #FFEEDB;
+  @extend %centre-flex;
+}
+
+.watch {
+  @extend %watch-size;
+  @extend %watch-radius;
+  position: relative;
+  background-color: #f9dc5c;
+  border: 16px solid #558564;
+
+  &__centre {
+  @extend %centre-flex;
+  @extend %watch-centre;
+  @extend %watch-radius;
+  flex-direction: column;
+  height: calc(#{$watch-size / 20});
+  width: calc(#{$watch-size / 20});
+  text-transform: uppercase;
+  background-color: #3185FC;
+  z-index: 5;
+  transform: translate(-#{$centre-absolute}, -#{$centre-absolute});
+}
+
+  &__minute-hand {
+  @extend %watch-centre;
+  @extend %clock-hand-centre;
+  @extend %animaton-time-directon;
+  background-color: #EFBCD5;
+  height: calc(#{$watch-size} / 35);
+  width: calc(#{$watch-size} / 3);
+  top: calc(50% - #{$watch-size} * 0.02);
+  left: #{$centre-absolute};
```

you don't need this braces (exception – when using vars in selectors like this `&__time--#{$index} {`)
```suggestion
  height: calc($watch-size / 35);
  width: calc($watch-size / 3);
  top: calc(50% - $watch-size * 0.02);
  left: $centre-absolute;
```